src/com/ok/Lesson4/Homework4/Main.java

```diff
@@ -0,0 +1,29 @@
+package com.ok.Lesson4.Homework4;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner myInput = new Scanner(System.in);
+        System.out.print("Input 1 - to see StackOverflowError or 2 - to see OutOfMemoryError: ");
+        int i = myInput.nextInt();
+
+        try {
+            if (i == 1){
+              new ProduceStackOverflowError().produceError();
```

Comment:
what is it?

Reply:
Can't get what's wrong with it ? Please explain.